### DISCUSSION TITLE
Documentation for script.js optional features

### DISCUSSION THREAD

**Discussion by dimejo** [plausible/analytics#4535] [Self-Hosted Support] [answered]
[The documentation](https://plausible.io/docs/script-extensions) for optional features of the tracking script previously had a section explaining how to adjust the script name to enable optional features. The documentation currently shows a screenshot with a checkbox menu to adjust the script - but those checkboxes do not exist in the community edition.

![Screenshot 2024-09-05 at 15-18-13 Plausible · Web analytics](https://github.com/user-attachments/assets/4ba6efbb-a128-4261-8a63-1c712d56842b)

Was the section of adjusting filenames removed to provide more value to the paid version of Plausible or did this happen by accident?

**Comment by ruslandoga** [comment, marked as answer]
👋 @dimejo 

AFAIK https://plausible.io/docs tracks Plausible Cloud which is the same as the `master` branch, so it might not be always correct with regards to CE releases, the latest of which is a few months old now.

**Reply by dimejo** [reply]
You are correct - https://github.com/plausible/docs/pull/533 was merged just 3 days ago.
